Monadical-SAS/monadical.com
# fix meta tags

fixed missing meta tags like preview image and description when sharing link with others 

To test:

- run locally `env FLASK_DEBUG=True ./server`
- `open http://127.0.0.1:5000`
- open dev tools and inspect page.. there should be some new metatags like below for all pages
-
![screenshot 2022-08-12 at 23 34 03@2x](https://user-images.githubusercontent.com/7106086/184417401-c69c926d-88c4-4ff2-b1c4-6e2f260c8893.png)
